src/renderer/entities/proxy/model/proxy-model.ts

```diff
@@ -1,5 +1,5 @@
 import { combine, createEffect, createEvent, createStore, sample } from 'effector';
-import groupBy from 'lodash/groupBy';
+import { groupBy } from 'lodash';
```

Comment:
What is the reason for this changes? Maybe better to use [this approach](https://tyapk.ru/blog/post/correct-way-to-import-lodash-methods) as we do in the most of places

Reply:
I think I had merge conflicts with your request so I was changing `dictionery` back to `groupBy`and webstorm imported it like that
changed back to how it was.